data/pki/meson.build

```diff
@@ -1,6 +1,11 @@
 # only install files that are going to be used
-supported_gpg = libjcat.get_variable(pkgconfig: 'supported_gpg', default_value: '0') == '1'
-supported_pkcs7 = libjcat.get_variable(pkgconfig: 'supported_pkcs7', default_value: '0') == '1' or host_machine.system() == 'windows'
+if libjcat.type_name() != 'internal'
+  supported_gpg = libjcat.get_variable(pkgconfig: 'supported_gpg') == '1'
+  supported_pkcs7 = libjcat.get_variable(pkgconfig: 'supported_pkcs7') == '1' or host_machine.system() == 'windows'
+else
+  supported_gpg = host_machine.system() != 'windows'
+  supported_pkcs7 = true
+endif
```

Comment:
the alternative way to do this is set `default_value` as `1` since we bumped the libjcat version _anyway_ the non-subproject one will support pkgconfig.

Reply:
Decided that's less magic. Switched.  Hopefully tomorrow or the day after a rebuild can be triggered and this all works :)